salt/utils/verify.py

```diff
@@ -432,8 +444,8 @@ def check_max_open_files(opts):
         return
 
     msg = (
-        'The number of accepted minion keys({0}) should be lower than 1/4 '
-        'of the max open files soft setting({1}). '.format(
+        'In file salt-master.service: LimitNOFILE={1} should be 4 times '
+        'the number of accepted minion({0}). '.format(
```

Comment:
It took me several hours to find out what the log message means with "max open files soft setting". I argue that this expression is Salt internal jargon, easily confused with the operating system max open file soft limits.

I finally found the setting by searching for the value of "16384" in git. 

I was under great pressure because the code says "the server will crash" and my minions where added fast by a software provisioning system (not Salt).

The objective of my PR is to give other not so bright sys-admins (like me) the most precise information how to change the potential loss of control, fast.
If LimitNOFILE is only used under Debian (or intitd systems) it could be guarded by a "if (Debian) then" block, and other contributors could add “other ways” according to OS.